src/main/java/uk/gov/companieshouse/itemhandler/itemsummary/DigitalOrderItemRouter.java

```diff
@@ -54,22 +57,9 @@ private void logItemGroupsCreated(final OrderData order, final List<ItemGroup> d
         sb.append("For order " + orderNumber + " created " + digitalItemGroups.size() + " digital item groups:\n \n");
         for (int i = 0; i < digitalItemGroups.size(); i++) {
             final ItemGroup ig = digitalItemGroups.get(i);
-            sb.append("\n + IG " + (i + 1) + " with kind " + ig.getKind() + " and " + ig.getItems().size() + " items:\n"
-                    + describeItemGroup(digitalItemGroups.get(i)) + "\n \n");
+            sb.append("\n + IG " + (i + 1) + " | " + ig + "\n");
```

Comment:
is this intentional?

Reply:
Yes, attempting to make these log messages more legible given they will be observed wrapped up in lots of JSON (for Kibana et al). Also relocated some of the logic to a new `ItemGroup.toString()` implementation.